cmsdb/processes/higgs.py

```diff
@@ -134,28 +171,34 @@
     name="tth",
     id=15000,
     label=r"$t\bar{t}H$",
-    xsecs={13: Number(0.1)},  # TODO
+    xsecs={
+        13: Number(0.1),  # TODO
+        13.6: Number(5.688, {
```

Comment:
```suggestion
        13.6: Number(0.5700, {  # value for mH=125 GeV
```
wrong, it's 0.5688 for 125.09 GeV. Additionally, I would recommend using the values for mH=125 GeV for consistency puposes and because the mass used for the generation is probably 125 GeV. I would also add a comment to say for which Higgs mass the value was taken (for bookkeeping)